debugger/DebuggerThread.hx

```diff
@@ -376,10 +364,10 @@ class DebuggerThread
             }
             // Allow at most 10K of classes in a message
             // Allow at most 100 classes in a message
-            if ((total == 100) || (f.length + byte_total) > (10 * 1024)) {
+            /*if ((total == 100) || (f.length + byte_total) > (10 * 1024)) {
```

Comment:
Why are you removing the limit on 100 classes and 10K worth of classes in a message?  This limit was put in place because the IntelliJ java VM cannot handle large messages.  It is possible that the limit was due to the recursive nature of deserialization of string lists in the code prior to this change, but it's also possible that it just can't handle large messages.